src/main/java/com/extrahardmode/config/RootNode.java

```diff
@@ -53,6 +53,22 @@
      */
     PRINT_COMMENTS("Print Comments", VarType.BOOLEAN, true, "If comments like this should be printed"),
     /**
+    * Globally disables certain modules. These settings might improve performance if these modules is not used. 
```

Comment:
Seems largely unnecessary, since you're (presumably) already checking the other configs to see if the feature is enabled in the first place?